fe/fe-core/src/main/java/org/apache/doris/catalog/JdbcResource.java

```diff
@@ -303,7 +303,7 @@ public static String handleJdbcUrl(String jdbcUrl) throws DdlException {
             // However when tinyInt1isBit=false, GetColumnClassName of MySQL returns java.lang.Boolean,
             // while that of Doris returns java.lang.Integer. In order to be compatible with both MySQL and Doris,
             // Jdbc params should set tinyInt1isBit=true&transformedBitIsBoolean=true
-            newJdbcUrl = checkAndSetJdbcBoolParam(newJdbcUrl, "tinyInt1isBit", "false", "true");
+            newJdbcUrl = checkAndSetJdbcBoolParam(newJdbcUrl, "tinyInt1isBit", "true", "false");
```

Comment:
I think we can delete this code if we do not need force convert tinyint(1) to be a boolean.

Reply:
This is to prevent manually setting it to true as this would affect our mappings